### ISSUE TITLE
Change required inputs message

### ISSUE THREAD

**Issue by DannyBen** [joerdav/xc#47]
First of all, thank you for this magnificent tool. It always makes me happy to see markdown and code combined together (I implemented a ruby gem that runs tests embedded in markdown :)

One minor suggestion:

When there is a required input that is not entered, the error message is:

```
Task has required inputs:
        xc hello [NAME] [HEIGHT]
        NAME=[NAME] HEIGHT=[HEIGHT] xc hello
```

I believe the square brackets are incorrect in this context, as they denote "optional".

Perhaps this?

```
Task has required inputs:
        xc hello NAME HEIGHT
        NAME=... HEIGHT=... xc hello
```

or, the below which creates a distinction between an environment variable name (CAPS), and a &lt;placeholder&gt; string:

```
Task has required inputs:
        xc hello <name> <height>
        NAME=<name> HEIGHT=<height> xc hello
```

Note that in this case, it is common to have the string between `<...>` in lowercase.

**Comment by joerdav** [comment]
@DannyBen thanks for the kind words, and the suggestion! ❤️ 

I like the second suggestion more. I'm open to contributions if you want to submit a PR. Or I can cover this off when I get around to it!

**Comment by DannyBen** [comment]
>  if you want to submit a PR

Since my Go is rusty (no pun intended), I'm gonna gave to say: "maybe" :)

**Comment by joerdav** [comment]
Happy to help if needed, I'll leave you to it but let me know if you give up :)

**Comment by joerdav** [comment]
Released in `v0.0.169`
